## Evolutionary Quantitative Genetics

Population genetics theory focuses on predicting what will happen to one or a handful of genes at most, whose alleles have discrete effects on the phenotype. But what of traits that vary continuously in populations (such as body size) and that may be encoded by many, many genes? Here, we will see how quantitative genetics extends the principles of population genetics to those cases, and gives us tools to understand and predict the evolution of phenotypes... sometimes without really needing to know the underlying genes.

### Duration: 

2 hours

### Script:

We saw that in population genetics, we could use some mathematics to extend our intuition about how a population will change over time, in terms of its composition with respect to alleles and genotypes at a specific locus, or a handful thereof. We did so by tracking genotype and allele frequencies, and this was possible if we know something about the fitness effects of the different alleles, their degree of dominancen or recessivity, the rate of mutations, the type of mating system, gene flow with other populations, etc. (Basically, if we know something about the evolutionary forces that apply to our population and how the loci under consideration affect fitness.)

Typically, we study the evolution of specific loci because we know they code for specific traits we are interested in. For example, we know that cystic fibrosis is caused by a mutated version of a protein called CFTR, or that HIV-resistance is conferred by a specific allele at another protein-coding gene called CCR5. Blood-type is another example of a trait that is encoded by one locus. Those traits are called Mendelian traits (and when they are diseases they are called Mendelian diseases), because their inheritance follows Mendel's laws, and so, they can be studied in a population genetics framework, because predicting the evolution of the trait here is equivalent to predicting the evolution of the gene that is causing the trait.

Unfortunately for us, most traits are not Mendelian. Think of body size for example. Body size is a quantitative trait. That is, there is not just two body sizes possible, small and tall, that correspond to one of two specific alleles, small and tall. Instead, in most populations we see a gradation of body sizes that seem to vary continuously: some people are between 160cm and 170cm, some are between 170cm and 180cm, and so on. Does that mean that there is a gene that codes for body size and that has many possible alleles, one for each possible body size? This seems unlikely, as patterns of inheritance of quantitative traits usually do not follow Mendelian genetics: if a tall person had kids with a small person for example, the offspring will tend to be of intermediate size on average (which suggests that there is a genetic component to it at least), but not always --- some of the siblings will be tall, other will be small, although they all received exactly half of their alleles from each parent. And because the inheritance pattern is not as clear as (say) cystic fibrosis or other Mendelian traits, it is therefore more difficult to find the genes that code for quantitative traits in the genome. And most traits are quantitative traits! Think of all kinds of morphometrics, but also the intensity of a color, flowering time, yields of milk and crop production.

Actually, most traits that humans care about in domesticated plants and animals are quantitative traits. Corn derives from wild varieties of maize with much smaller yields. Cows come from aurochs, that have been selected for their behavior and producing more milk. Wolves have been domesticated into dogs, and their morphology has been shaped into all the different canine races we see today. In all those examples of domestication, humans have applied selective breeding to evolve those quantitative traits they were interested in, to produce crops that yield more and more, and dogs with smaller and smaller snouts. Only allow smaller dogs to breed, and the population will become smaller (chihuahuas are actually much smaller today than they supposedly were when Europeans first documented them upon arrival in the Americas).

So, how good of a theory of evolution population genetics really is, if it can only make predictions about the evolution of traits with a "simple" Mendelian genetic basis, but not the great majority of phenotypes, which are quantitative traits? And that is what we are generally aiming to explain with a theory of evolution: the evolution of phenotypes (why do specie change through time?). By means of the evolution of genotypes, yes, but nevertheless the ultimate goal is to explain the evolution of traits.

Well, what the above examples about domestication (e.g. that traits will change with selective breeding) suggest is that evolutionary forces that we identified in population genetics also somehow apply to quantitative traits, which is good news. We just have to figure out how. So how do we go about that?

The first major insight we should probably have is that just like positive selection for an allele will increase the frequency of that allele in the population in a population genetics framework, the change in the quantitative phenotype of a population that undergoes selective breeding means that some "unit of inheritance" affecting the value of that trait is passed on down the generations, just like an allele. In other words, that trait is heritable. So what is the unit of inheritance for such a quantitative trait? Well, here the insights from Gregor Mendel apply: when Darwin (who had no idea about genes) supposed that whatever was the unit of inheritance of quantitative traits from both parents would "blend" together when they reproduce to form an offspring (thus explaining why hybrids between species often have intermediate phenotypes, show an example of bird-of-paradise), Mendel proposed that inheritance was particulate (not continuous), and that discrete genes were the support of it. But wait, we just said that unlike pea color and texture, most traits do not follow the classic Mendelian laws of inheritance! How could traits that vary continuously in a population be encoded by discrete genes?

Well, let's look at the following example. [Illustrate a genotype-phenotype map expanding to give rise to a continuous distribution in a population.] And so, we see that when a trait is encoded by many genes with relatively small effects, this can produce a nerly continuous distribution of phenotypes in a population, even if the unit of inheritance is discrete. Quantitative traits may just be encoded by many genes! Because of that, they are also sometimes called "complex" traits. 

Note here that this model predicts that the offspring of a single pair of parents will be diverse if there is recombination, which is what we observe in most quantitative traits. As we mentioned before, an alternative hypothesis for what might be the genetic basis of those traits is that they could be encoded by a single gene, just with lots of "continuous" alleles (one for each body size essentially). But this would produce only one type of offspring for a given cross, which does not match observations. So, many genes for a trait seems to be a better explanation than one gene, many alleles for a trait. [Maybe a small primer on recombination?]

So, we figured that quantitative traits were probably encoded by the same mechanisms as Mendelian traits, just more genes. If they are encoded by discrete genes with discrete alleles, does that mean that the laws of population genetics also apply to them? The answer is yes, but the theory needs to be expanded a little bit to now account for a multitude of genes.

In designing a theory of how traits evolve, we are usually interested in the effect of selection, as it is often recognized as the most important "driving" force behind evolution. So, can we expand the framework of population genetics to predict how a trait will evolve under selection? This is where the theory of quantitative genetics starts really. If you've noticed, in the simulation we used to illustrate the "polygenic" nature of quantitative traits, we saw that the distribution of traits generated in the population was not only continuous, it also looked like a normal distribution (i.e. a bell-shaped, or Gaussian, curve). (It is actually a binomial distribution with lots of discrete bins that ends up approximating a normal distribution as the distance between bins becomes infinitesimal.) Why do we care, you might say? Well, normal distributions are great, because their mathematical properties are very well figured out. And so? So, we can apply mathematical tools to predict what will happen to this normal distribution --- this is the core of quantitative genetics theory. Here, we won't go into too much detail about how the laws of quantitative genetics are derived [maybe have some appendix for that], but we will see the kind of insights those mathematical treatments came up with.

Also remember that the quantitative traits we are concerned with here are "complex" traits: they might be encoded by lots of genes, but also and most importantly, we do not know what those genes are (not like cystic fibrosis) --- we just have to assume they might be there in the genome. This means that quantitative genetics is in its very essence a theory of phenotypic evolution, in that it aims at making predictions about the evolution of quantitative traits without actually knowing the exact underlying genetic basis of those traits (using some mathematical properties of some assumed model for how genes code for the phenotype --- e.g. that phenotypic variation will be normally distributed). 

Note that this expectation that a variable such as the phenotype coded by many genes will be normally distributed is more pervasive that one might think, and actually applies to lots of variables (not just the distribution of phenotypic effects of genes). We shall see other examples where a normal approximation is justified later, but for now just bear in mind that this "trick" is used a lot in quantitative genetics, because it allows to derive models, laws and eventually predictions. And fortunately for quantitative genetics, normally distributed variables are expected to occur a lot in nature, because of something called the central limit theorem, which states that the sum of random variables will be normally distributed, even though the random variables themselves might not be normally distributed. This may seem counter-intuitive, but here is a small simulation that illustrates it. [Show the simulation.] The central limit theorem is one of the most important theorems in probability theory, because it has a lot of handy consequences for statistical applications in the real world (such as that normal approximations are okay).

So, is there a law that predicts the evolution of a quantitative trait under selection? Yes, there is, and without further ado, here it is: R = h^2 * S. This is the basic equation of quantitative genetics, also called the breeder's equation. So what does it mean? [Show a distribution of phenotypes and explain.] It means that if we selected only one portion of the population to mate (like a breeder would do), then the mean of the population would change by a certain amount given by R, the response to selection, and that depends on the selection differential S [show on the graph what that is], and on the heritability of that trait h^2. 

We mentioned heritability before. Darwin already figured that in order to evolve, a trait had to be heritable (that is one of the conditions for evolution by selection to take place, together with the variation in trait in the population and a link between trait and fitness). And of course, we know that the basis of inheritance (in most cases that are relevant for the theory here) is mostly DNA and genes. But note that here, heritability is a value (which actually ranges between 0 and 1), meaning that it can vary from one trait to the other. How is that possible? After all, if a trait is encoded by some genes, whether it is one or many, then it should de facto be heritable, right? Like, you get diabetes, cystic fibrosis or your body size from your parents. Well, not exactly. For one, there are traits that are not determined by genes at all: for example, whether you know English depends on whether you have learned English in your life --- no-one possesses an allele that codes for knowing English. You might also have higher levels of cortisol (the stress hormone) in your bloodstream right now compared to your neighbor, and that may be because you just experienced something stressful that your neighbor didn't. Hence, not all phenotypes are determined by genes, many are actually determined by experience, learning, developmental history and external circumstances. In quantitative genetics lingo, we say that those traits are determined by "the environment" (basically anything that is not the genes). They are also called "plastic", as they vary due to changes in the environment and not genetic changes (but there will be a lecture about phenotypic plasticity).

[Talk about the invalidated marshmallow test?]

So, those were traits that may not be genetically determined at all (whether and how they will "evolve" is largely out of the scope of quantitative genetics therefore, h^2 = 0). But traits can also be (and often are) partly genetically and partly environmentally determined (and this can apply to discrete as well as quantitative traits). For example, it is well established that there are genetic predispositions for type II diabetes (if it is rampant in your family you may carry genes that increase your chances of getting it), but that lifestyle is also a major risk factor (sedentarity, smoking, overweight and too much sugar and fat in the diet). This is an example of a partly genetic, party plastic trait. And so, because selection affects the spread of inherited particles, to know how much a trait will change under selection we need to know how much of it is heritable, and how much is not. This is why we need to know the value of h^2.

So how do we go about that? Well, this is where the normal approximation comes in handy. Why? Because it allows us relatively basic statistics such as linear regressions to come up with an answer (and actually to come up with the main equation we saw above, as we shall see). So how do we know how much of a trait is heritable? One way is to measure trait values in many individuals and compare them with the same trait measured in their offspring. The more similar parents and offspring are, the more heritable the trait is. This can be done by plotting offspring phenotype against parental phenotype for a given population sample. Of course, each offspring has two parents and each parent can have a multitude of offspring, so for statistical independence purposes it makes more sense to plot what we call the "mid-offspring" phenotype against the "mid-parent" phenotype, that is, for each mating pair of parents, the average trait in their offspring versus the average between the two mating partners. We can then perform a basic linear regression, i.e. draw a best-fit line through this cloud of points. And well, it turns out that the slope of that line is the heritability, h^2: the higher it is, the more parent phenotype is a good statistical predictor of offspring phenotype and so, the more heritable the trait is. In contrast, if the line is flat, and h^2 is close to zero, it means that there is little-to-no correlation between offspring and parent phenotype: the trait has poor heritability.

So, we can estimate heritability by using a linear regression of offspring against parents, and feed it into the basic equation we saw. But where does that equation come from anyway? Well, the answer is also in that linear regression. [Show on the plot.] When we apply (directional) selection, we are essentially allowing only part of the population to reproduce, or contribute to the next generation. Let's call the mean trait value of this subset of reproducers P', while that of the whole population is P. Well, based on our fitted linear regression and its slope h^2 we know that we can predict the new mean of the offspring of that subset of reproducers, O'. How that differs from the mean trait value that the offspring would have had without selection, O, is the response to selection: how the trait has responded to an episode of selection in the next generation, and so R = O' - O. Similar, the selection differential is the difference in phenotype between those that have reproduced and the mean phenotype of reproducers if there had not been selection, so S = P' - P. And from basic geometry we know that (O' - O) / (P' - P) = R / S is the slope of the line, h^2, so that h^2 = R / S, and so R = h^2 * S.

Tadaa! So, quantitative genetics predicts that the change of a quantitative trait under selection will depend on the intensity of that selection pressure and on how heritable the trait is. Note that for the demonstration, we used the simple case where an entire subset of the population is completely prevented from breeding (like a selective breeder would), but the theory still holds when individuals have more (rather than all-or-nothing) chances of reproducing as we move along the range of trait values. But wait, in this example we explicitly assumed that one end of the continuum was "fitter" than the other. That is, we assumed directional selection. We may have seen multiple types of selection in previous lectures on population genetics. Here, it is important to described new terms and make important distinctions about lingo.

At the genetic level (i.e. when looking at the fitness effects of single genes or alleles), it is common to classify the forms of selection into positive selection, negative or purifying selection, balancing selection, overdominance or underdominance. At the phenotypic level, and especially for quantitative traits that cannot simply be boiled down to a single locus, we use different terms. Instead, we talk about directional, stabilizing or disruptive / diversifying selection, depending on the range of values of the trait that are favored relative to the ones that are disfavored. Being aware of this distinction of scales is important, because there is no direct connection between the forms of selection at the two scales, and this can lead to confusion. For example, geneticists often talk about stabilizing selection to refer to cases of heterozygote advantage (overdominance), because it seemingly "stabilizes" the encoded phenotype to some intermediate between the two homozygotes, whose advantage would then be called "directional" (because then the two extreme phenotypes would be favored). However, one can show that "true" stabilizing selection on a quantitative phenotype encoded by many genes (i.e. one optimal value along a continuum) could just as well lead to the fixation of homozygous genotypes at many genes, or heterozygous genotypes at many genes, or a mix of both (some genes where heterozygotes are advantages and others where homozygotes are advantaged). Point being: bear in mind that the terms we use to refer to different forms of selection may have different meanings depending on whether we are talking about phenotypes or genotypes --- so, when handling those concepts, make sure you are clear with what scale you are at (this may save you some confusion and sterile arguments). [Maybe a little simulation?]

All of this to say that here we restrict ourselves to the terms directional, stabilizing and disruptive, in terms of phenotypes (cause that's what quantitative genetics is about --- it should probably have been called phenetics). So what happens under other types of selection? Well, quantitative genetics also has it figured out: stabilizing selection will restrict the variance in phenotype in the population (i.e. "narrow down" the distribution) while disruptive selection will spread it more (and so increase diversity). What is key to predict the response of the population distribution to different forms of selection here is the shape of the "fitness function", i.e. how traits determine fitness. If that is figured out relatively precisely, the outcome of selection can be predicted. [Maybe a little simulation?]

Many predictions of quantitative genetics have been verified empirically, in artificial selection experiments involving model systems such as Drosophila, livestock or crops (and for the two latter there is a big incentive to test those predictions). However, sometimes the breeder's equation is wrong. There are many ways in which it could be, but here we focus on the basic fact that selection does not necessarily always act on single traits. Organisms are composed of multiple phenotypes, many of which have been shaped by various selective pressures and biological constraints over millions of years. By selectively breeding a given organism, it is possible that we hit some limit --- for example, although we have bred dogs to become as large as great Danes and as little as chihuahuas, we will probably never see dogs as large as T-rexes and as small as an ant, no matter how strongly we select the breeders. In a sense, while we impose some selection for more extreme body sizes, there is an antagonistic selection pressure for increased viability that starts to kick in whenever body sizes become truly extreme. 

The response of one trait to selection may also depend on the selective pressure applying to other traits, if those traits are genetically correlated (i.e. meaning that a genetically-determined change in one will cause a change in the other). For example, it could be that genetic changes causing longer beaks in some birds also cause the beaks to be wider. This source of genetic correlation, where the same genes code for different phenotypes, is called pleiotropy. If that happens, then we would make mistakes in predicting the response of beak length to selection if we do not consider selection applying to beak width, and the genetic correlation between both traits. [Show example.] Imagine a two-dimensional phenotype space, where the third dimension may represent fitness and gives us and idea of where directional selection is pointing. This is called a fitness (or adaptive) landscape, and according to our basic evolutionary intuition, we may expect selection to "push" a given population up towards the top of a hill in that landscape (some "local optimum"). Note that this fitness landscape is another way to call the fitness function we have mentioned before, except here fitness is a function of multiple traits, not just one. Also note that fitness landscape can have many more dimensions that two, but of course they would be impossible to draw. One prediction of quantitative genetics in that case is that the population may not necessarily evolve in a straight line towards a fitness peak, but may get there through a route that is constrained by the genetic correlation between the traits under selection. As per Darwin's insight that selection needs variation in a trait, evolution will preferentially follow the direction of largest genetic variation (and there will be a clear such direction if the traits are closely correlated). We say that evolution follows the "lines of least genetic resistance", highlighting that the amount of genetic variation really consistutes a constraint to the response to selection. This all part of the theory about multivariate traits is encapsulated in the so-called multivariate breeder's equation, Delta zbar = G * beta. Delta zbar is a vector of change in mean phenotype, for each trait considered (it is the multivariate equivalent of R in the breeder's equation). The so-called G-matrix is a matrix of genetic variance and covariance among those traits (it summarizes how much they are genetically correlated). It describes something akin to heritability in the classic breeder's equation. Finally, beta is a vector of directional selection gradients (the coordinates of the "selection pressure" vector shown in the multidimensional fitness landscape). It is the multivariate equivalent of the selection differential S in the breeder's equation.

Another prediction of quantitative genetic theory is that selection works on "additive" genetic variance. What does that mean? Remember the basic model we used to show that many genes with small effects could give rise to a continuum of phenotypic values? Well, that model was an additive one, because we did sum up the values of each allele at each gene to come up with a trait value for each individual. In other words, we assumed that all loci had additive effects on the phenotype. But there are other things that can affect the value of a trait besides the additive effects of its genes. For one, we already saw that the environment, as opposed to the genes, can affect the value of a trait. In quantitative genetic models, the value of a trait for one individual is often decomposed as P = G + E, where G is its genetic value (not to be confused with the G-matrix), i.e. how much of the phenotype is determined by its genes, and E is the environmental effect, i.e. how much is determined by the environment. 

But there are different types of "genetic effects". Additive effects, as we have described already, are those effects that stem from the independent cumulation of contributions from different loci. But we also have dominance effects, where the fact that the dominance of some alleles over others at some loci will introduce deviations in the distribution of phenotypes from what we would expect if the alleles were co-dominant. And we have epistatic effects, where some alleles are simply not independent from the effect of alleles that occur at other loci. That is, under epistasis, the effect of an allele on the phenotype of an individual depends on which alleles it is found in combination with (those other alleles are called the genetic background of the focal allele). Dominance effects and epitatic effects are often termed non-additive genetic effects: they are effects of the genes on the phenotype, just not additive.

The variance of a phenotype in the population is often described as (using some approximations): Vp = Vg + Ve, where Vg = Va + Vd + Vi. This variance decomposition simply describes what we said above: that the amount of phenotypic variance observed is partly explained by differences in environment and differences in genes, and that the part that is explained by differences in genes can be further split into a part attributable to differences in additive genetic differences, a part that is explained by dominance-recessivity relationships and a part that is due to epistatic interactions. Note that we are building a conceptual model here: we are dealing with a hypothetical trait and we are assuming that it is caused by all these hypothetical effects of genes we do not know. What we are saying here is that in a hypothetical case of a trait that satistfies those assumptions, the above equality should more-or-less be true. Okay fine, but what do we do with that? Well, that sort of mathematical exercise was used by Ronald Fisher to come up with his "fundamental theorem of natural selection", which states, as we said above, that the rate at which the mean fitness of a population increases is proportional to the additive variance for fitness (if we count fitness as a phenotype). A related insight is that the rate of change in a trait is predicted to depend on the portion of genetic variance that is attributable to additive effects, and not to dominance and epistatic effects. 

But how is that possible? After all, we saw in population genetics that the change in frequency of alleles that are dominant or recessive is quite different from alleles that are co-dominant, right? And if the effect of an allele depends on its genetic background of interactions with (possibly) many other alleles at many other genes, that should also affect how this allele will change in frequency, right? That is correct, but the rationale behind Fisher's insight is that although dominance and epistatic effects are genetic and not environmental, they are not as heritable as additive effects. An additive effect is, by definition, an effect of an allele that only depends on that allele. For example, an allele that has the additive effect of increasing blood pressure by 2% will increase your blood pressure by 2% as long as you have it, irrespective of other alleles at the same locus (dominance) or at other loci (epistasis). Now an allele with a dominance effect on your blood pressure may increase your blood pressure by 2% only in homozygous state, if it is recessive, for example. What effect that allele will have in the next generation will therefore depend on what allele will have been inherited by the offspring from the other parent, so, it is less predictable what will be the contribution of that allele to the offspring phenotype (its effect is less heritable). Similar, if an allele increases your blood pressure by 2% only when in combination with X other alleles at X different loci in the genome, it is very hard to predict the effect this allele will have at the next generation, because we don't know in which genetic background it might end up. Of course Fisher worked this out mathematically but the crux of the rationale behind his argument is that non-additive effects are not very heritable, especially in a genetically diverse population where a given allele may be found in combinations with a variety of alleles at its own locus (dominance) and at other loci (epistasis), and so those effects may not be acted on efficiently by selection. In contrast, the additive effects of genes are consistent across generations, and so the response to selection won't just depend on genetic variation, it will depend specifically on additive genetic variation. 

And this has real world implications, for example for breeders: if you want to know how much a trait can change in a population, you better know what the additive genetic variance is, not just the phenotypic variance. Of course, in quantitative genetics we always assume that in practice, we don't really know which genes code for the trait we are trying to evolve, so how can we calculate the variance in additive effects of genes? Well the good news is that there are ways around that: because additive variance can be viewed as the portion of phenotypic variance that is truly heritable, many use the equality Va / Vp = h^2. And so, if we can measure heritability h^2 (e.g. using offspring and parent regression) and the variance in phenotype (that should be easy), then we can in theory estimate the amount of additive variance in a trait.

But let's rest a bit on some weird assumptions behind those quantitative genetic predictions. Is it really true that an allele whose effect depends mostly on epistatic interactions will have no consistent effect from one generation to the next? That would be the case if the allele is found in different genetic backgrounds, but how often does that really happen? After all, populations tend to be of finite sizes, they also tend not to be immensely diverse genetically, and individuals tend to mate within local populations in many species, so it may not be so weird to think that more often than not, an allele will be found in more-or-less similar genetic backgrounds (i.e. more-or-less in combination with the same alleles at other loci) as it passes through the generations. And if that's the case, the effect of that gene may be quite consistent through time and across the population. Would it then not matter just as much as alleles with additive effects for how the trait it codes for will respond to selection? In other words, are we not making a mistake by only focusing on additive variance? And besides, what are additive effects really? Our whole theory so far relies on the premise that there are genes with fixed effects on the phenotype that add up. But as we (hopefully) saw in population genetics that genes cannot always just be attributed constant effects on fitness (e.g. when selection on a gene is frequency-dependent), what if additive effects actually didn't exist? Yes they are very handy because they can produce continuous distributions of phenotypes and allow us to use nice mathematical tricks to make predictions, but we know a thing or two about molecular genetics and how genes actually code for phenotypes: and that often involves signalling cascades, metabolic pathways, transcription regulation and integrated developmental processes --- all kinds of cellular mechanisms that require interactions (and so, epistasis) between many different genes. To the point that genotype-phenotype maps are consensually best represented by gene regulatory networks than by genes adding additively in forming the phenotype. Actually, (and still because of the central limit theorem) one can easily obtain a normal distribution of phenotypes by modelling a trait as being encoded by a gene network with many interactions, instead of genes with additive effects. So, why does quantitative genetics predict that additive variance is so important, if it is so hard to even put the finger on what those additive effects actually are?

Well, that is because additive effects are more of a statistical, rather than a functional, concept. What does that mean? It essentially means that additive variance may capture the influence of functional (i.e. "real") genetic mechanisms that are not necessarily allelic effects independently adding up. To illustrate that, we need to have a look at what additive variance really means, that is, how it is theoretically computed. Imagine a trait encoded by many genes. Now, let's focus on one of those genes --- we are in a hypothetical case where we assume that we know that the gene acts on the trait. We further assume that we can genotype every individual for that gene in the population, and that is has two alleles segregating into three possible diploid genotypes: aa, Aa and AA. Let us also assume that this gene contributes for the trait in ways that involve some level of dominance and some interactions with other genes, but we don't know how exactly. We do not know if the gene has direct, additive effects on the phenotype. What we know, however, is the genotypic value of each individual: that is, the genetically-determined part of the phenotype that is accounted for by that one gene. Of course, if some of the effect of the gene is through interactions with other genes we can hardly count that as being accounted for by that one gene, but we will assume here that any interaction involving that gene counts as part of its genotypic value. (In practice we don't really know the genotypic value of a gene for a trait in some individual, all we know is the phenotype of the individual, but bear with me for now --- remember, this is a hypothetical case.) So we assume that we have collected information about the genotype and genotypic values at that one gene for all individuals in the population. We now plot them. [Show the regression plot as we explain.] How is the "additive" effect of that gene computed? It is computed by drawing a regression line through this cloud of points. The line represents the phenotype (or genotypic value rather) that every individual would have if the gene was coding purely additively for the phenotype. The predicted value of each genotype based on additivity (or rather its deviation from the mean of the population) is called the breeding value of that genotype. The slope of the regression line is the average effect of an allele substitution at that gene: when one a-allele is replaced by one A-allele, this is how much the phenotype changes on average in that population. Now, we may notice that the mean phenotype for each genotype deviates from the additive expectation. That deviation is the dominance deviation of each genotype. For example, the heterozygotes may be closer in phenotype to homozygotes AA than to aa (something that is not capture by the straight regression line). Finally, each individual may deviate from the mean of its genotype group. In the absence of environmental effects (which we assume for simplicity for now), this deviation is an epistatic residual, i.e. individuals differ from what should be expected for their genotype based on additivity and dominance, because their genotype for that gene is found in combination with different genetic backgrounds in each of them. Now, the variances in breeding values, dominance deviations and epistatic residuals across the population are what we referred to as additive variance Va, dominance variance Vd and interaction variance Vi, respectively. That is the additive variance that Fisher says matters for the response to selection. 

And so you see: these variance components are not calculated by actually looking at whether the gene does act independently from other genes, in its molecular mechanism, when coding for the phenotype. Instead, we remain very agnostic about the true molecular mechanisms at play, we only partition the phenotypic variance we observe into bins (additive, dominance, epistasis) that exist in a phenomenological model that we have created as an approximation of reality. This is not to say that dominance and epistasis don't exist in reality --- they most certainly do, but this is to illustrate that those variance components are computed based on statistical variance partitioning, and not based on actual biological mode of action. So what? Is it bad? Well, not necessarily. It means that any molecular mechanism that would cause one genotype to have on average a different phenotype than some other genotype, may be counted as a (statistically) additive effect (because it will cause the regression line to be bent), even though additivity may not be the functional mechanism at play. Consider an allele that has increases blood pressure by 2% only in a specific genetic background (e.g. because its encoded protein has to interact in various complicated ways with other proteins). Well, what if selection has shaped that genetic background during millions of years such that most individuals share the same combination of alleles (the combination that works). In such a population, nearly all individuals with genotype AA may have a blood pressure 2% higher than Aa individuals, who also are 2% higher than aa individuals. Just because those allele substitutions occur in an otherwise relatively homogenous genetic background with little genetic diversity. And even though the gene inherently works through epistasis.

So, epistasis can feed into breeding values and additive variance, because it can "look like" additive genetics at play. But then, what is the bit of variance that we attribute to epistasis (Vi), if the epistatic effects of the gene have already been counted in additive variance? That is the difference between functional and statistical epistasis. Functional epistasis refers to epistasis in the mode of action of a gene: it actually interacts with other genes. But as we saw, this may look additive and be captured by additive variance. Because this variance partitioning process of quantitative genetics is inherently a stepwise process of encapsulation of deviations from expectations, we have to end up with residual variance at the end. That residual variance is the interaction variance Vi: the variance in phenotype that did not look like additive effects, or like dominance effects, and that is therefore relayed to being epistatic in origin. That is statistical epistasis. In the absence of environmental effects, statistical epistasis is probably truly caused by functional epistasis (i.e. individuals probably differ in residual deviation because of differences in genetic backgrounds), but not all of functional epistasis will end up captured in statistical epistasis (actually with high probability a good part of it will be captured by additive variance).

Great, so those variance components are statistical concepts that can encapsulate different genetic mechanisms: additive variance could be caused by true additive effects, but also by epistatically-acting genes that are just found in a similar genetic background. This makes additive variance sound like a weird, artificial mix of multiple genetic mechanisms that have nothing to do with each other, so why is it such a big deal (remember, the fundamental theorem says it is all that matters for the rate of the response to selection)? That is because of one thing those effects have in common: they are heritable. We saw that epistatic effects could be heritable if they occur in a more-or-less similar genetic background. Therefore, those are the epistatic effects that selection can act on. It makes sense, then, to separate epistatic effects into those that are heritable and those that are not, if what we are after is the response to selection. With that respect, heritable epistatic effects are much more similar (in their evolutionary consequences) to true additive effects than to statistically epistatic effects. So, this whole separation of the variance into additive and non-additive is mostly about separaring the heritable sources of genetic variation from the non-heritable ones (remember that h^2 = Va / Vp). Additive variance does not necessarily mean variance in effects that are additive, it is the variance in effects that look additive, and that therefore are heritable, and that's what matters because that is what selection will work with. And as far as we are concerned, true additive effects may not exist at all (genes may only be acting through interactions with other genes) --- that would not prevent the partitioning of variance into additive and non-additive to be a useful tool for predicting evolution.

With one thing to keep in mind though. Quantitative genetics makes predictions about the evolution of traits based on some partitioning of phenotypic variance to isolate the portion of it that is heritable (i.e. additive variance). It is important to note that heritability is contextual. Like we said: some epistatically-interacting allele will have a heritable effect on the phenotype if the genetic background remains more-or-less the same. If that background changes and becomes genetically diverse (e.g. new migrants come into the population from elsewhere, or the environment changes, favoring different alleles at the interacting genes), that allele will end up loosing a lot of its statistically additive effect. In the regression we used to partition variance, the slope of the regression (the average additive effect of an allele substitution at that gene) will become close to zero. Why? Because variation in the genetic background will bring forth lots of combinations of alleles, with all kinds of different phenotypic effects, so that a change in allele at that gene no longer has a consistent effect: that effect will be very noisy, depending on which background the allele is found in, and most of the variance in phenotype attributable to that gene will fall in the residual, statistically epistatic variance component Vi rather than in the additive variance component Va. The trait will have lost a lot of the heritability that was once attributable to that one gene. 

This means that as populations evolve and (e.g. environmental) conditions change, so do the quantitative genetic basis of traits. The heritability of a trait may change with time and with changes in the genetic composition of the population. Another example is that a trait that may be influenced by the environment will seem highly heritable and genetically-determined if the environment remains constant or rather homogeneous: if the environment becomes heterogeneous, differences between individuals will rise due to environmental reasons and heritability will drop. For example, the incidence of type II diabetes among a fictional population of people with very similar lifestyles and eating habits (a sect maybe?) will appear highly heritable and genetically determined, just because the source of environmental variation was removed. And so, heritability is a concept that is relative to a certain population, in a certain context, at a certain time and with a certain genetic composition. And this has a big consequence on quantitative genetics as a theory of evolution: it is inherently a theory that predicts short-term evolution. Empirical results are consistent with that: the theory predicts relatively well what happens in experimental lines over a few generations, but falls short of explaining longer-term evolution, where the rules of the "statistical" genetic basis of traits basically changes. In other words, while its reliance on statistical estimations of genetic effects makes quantitative genetics a powerful framework to predict the course of short-term evolution without having to know much about the actual genetic underpinnings of traits, this same advantage predicts its own downfall, because those statistical estimates are good proxies for genetic effects only given the current circumstances of a population.

That's grim. But quantitative genetics has led us a long way in understanding how evolution proceeds, and in connecting the principles of genetics to what Darwin was originally intersted in: explain how traits evolve. It has also provided tools to "improve" domesticated plants and animals. Breeders have been breeding animals for a very long time [find examples of when different things were domesticated], and it worked, without them ever having to know anything about quantitative genetics. Just let the smaller dogs breed, and the population will become smaller. However, that theory provided a solid framework to do so much more efficiently: now breeders measure heritability and additive variance, use the theory to predict what crossing schemes may be the best, and screen individuals for genetic markers associated with certain traits to achieve the desired results.

Let us finish this lecture with that one last thing I mentioned: screening for genetic markers. For the whole lecture we were saying that quantitative genetics theory concerns itself with predicting the evolution of phenotypes, without having to know the actual genes coding for a trait. Well, the field has been expanded to provide tools to actually look for those genes that may be involved in a given trait. (One might argue that those tools have actually been provided by the field of genomics than quantitative genetics, but it still relevant to mention them here.) So, how do we know what genes may contribute to a quantitative trait? By looking at statistical associations between genotypes and phenotypes, assuming that we have the means of genotyping individuals for particular genes. What we are after is loci whose alleles correlate with a change in phenotype: that is [show the regression again], we are looking for alleles with seemingly statistically additive effects on the phenotype. (Remember that all the caveats we mentioned apply here: those genes may appear involved in the trait now but may lose their additive effects in the future, and there may also be loci involved in the trait through lots of noisy epistatic interactions that just won't show up because they have no statistically additive effect.) So, it is relatively safe to say that the methods that look for these types of statistical associations are after loci that contribute to the additive variance in a trait. (Actually some look at possible interactions between alleles at different loci, so are aiming at detecting epistasis.) These methods come in different flavors. One of them is QTL mapping. A QTL (quantitative trait locus) is a locus, or molecular marker somewhere in the genome, that may be statistically associated with the trait under consideration. Given a number of known markers that can be genotyped in a sample population, usually of related individuals, one can look at what markers correlate most with variation in the trait. Those markers may be close to loci that have an effect on the trait. Another popular class of techniques is GWAS (genome-wide association studies), where the whole genomes of many unrelated individuals are sequenced, single nucleotide polymorphisms (SNPs) are identified, and are correlated with variation in the trait. The spirit is similar, it is just that one uses families and specific sets of markers and the other one a lot more genetic data and requires a large population. With this type of techniques, it has been confirmed that many quantitative traits indeed involve lots of genes with relatively small (statistically) additive effects.
