## Tree thinking

In this lecture we will learn about phylogenetic trees: what they are, why they are so important to evolutionary biology and how to build them. In the accompanying practical, we will give you a more hands-on idea of the various applications of phylogenetic trees.

### Duration

1 hour (+ 3 hours of practical)

### Script

* In this lecture we will focus on one of the most used pieces of data in evolutionary biology: the phylogenetic tree. We will learn what it is and what it represents, how to build one and what kind of things we can do once we have one.

* So what are phylogenetic trees? Just like genealogies show family relationships among people, phylogenetic trees show "genealogical" relationships among lineages, taxa, or species, in a graph that has the shape of a tree. Of course, this is assuming that there are such relationships between species or lineages, and that premise comes directly from Darwin's insight that all species are related. If you compare species anatomies or genetic sequences or behaviors or whatever traits, you will find differences but you will also find lots of similarities. [Give the example of the arm.] If descent with modification (i.e. evolution) is true, then similar but not exactly identical structures in different species may very well be the consequence of those species being genealogically related ---- they share a common ancestor somewhere in the past (ofc, much further away in the past than any family genealogy). We then talk about characteristics that are identical by descent in this case. And all of this means that phylogenetic trees are actually genealogies between groups of organisms, just much more distantly related than your family members. If you go back the generations long enough, you would get about 6 Myr ago to an organism that was some sort of intermediate between humans and chimps. Part of the descendents of that organism led to humans and another part led to chimps and bonobos.

* So how do we read a phylogenetic tree, or phylogeny? The branches basically represent degrees of genetic relationship between lineages. [Show example of primates.] [Introduce the concepts of nodes, branches, tips, root, time, most recent common ancestor, and ofc the fact that some lineages are more closely related than others.] [Also make clear that it is often, but not always, groups that we considder in phylgoenies in contrast to genealogies where we track individuals.] [Maybe talk about the differences with family genealogies --- we don't have all the ancestors, and we go "outwards".] [Also add fossils and say that fossils are not necessarily THE ancestor --- maybe use dinosaurs and birds and archaeopteryx for that.] 

* So how do we build a phylogeny? Remember something very important: with those tree graphs we intend to represent genealogical relationship. We could have made trees in whichever way we want, e.g. putting everything that we eat, that is useful, that is a pest, or that swims, together for example. The rules that we use to clump taxa together depend on what we eventually want to achieve with this type of dendrogram (tree-like graph). A phylogeny is not any dendrogram: it is a dendrogram that is supposed (if we built it correctly) to tell us something about the evolutionary relationship between lineages. And that means that to build it we need to use criteria that are informative about those relationships. And that's where it gets thorny.

* Naively, we could use the principle of resemblance to infer that two species are closely related. After all, people who are more closely related tend to resemble each other more, and we could argue that humans and chimps resemble each other much more than either resembles a fruitfly, so they must be closely related. That is a good intuition, but there is a trap: not all similarities are identical by descent. Some features of organnisms are shared, or similar, for other reasons which may have nothing to do with having been inherited from the same ancestor. We call those homoplasies. The ones that are identical by descent we call homologies. Homoplasies are common features that were acquired independently, for example, the streamlined shape of marine mammals and fish, or the ability to fly. [Maybe give the example of vision in cephalopods versus vertebrates or cognition in birds versus mammals.] And it makes sense that homoplasies would arise: species evolve under natural selection and adapt to their environment. So, it is not too surprising that when very distantly related lineages experience a similar environment, they will evolve similar ways to cope with that environment. That process is called convergent evolution (sometimes parallel evolution but that means something slightly different), and is a primary cause of homoplasies. But similar selective constraints are not the only thing that cause homoplasy --- they can arise under pure chance as well. Imagine a DNA sequence, there are only a few ways that bases can change, so there could well be a random subsitution in two distantly related lineages.

* This all means that if we want to use resemblance in traits to infer genealogical relationships, we need to filter out homoplasies and focus on traits that are identical by descent. And that begs the question: how do we know if a trait is identical by descent or if it is a homoplasy, if we do not know the genealogy of the spceies (we would need that to tell apart the two, but that is precisely what we are trying to reconstruct). There are ways to get out (not with 100% certainty but with some confidence) of this vicious circle. For one, we know that some traits may be more amenable to be shaped by natural selection. The general shape of the body, for example, seems like an obvious candidate of something that would change if an organism evolved from a terrestrial to an aquatic lifestyle. Same for wings. But the inner composition and constitution of the limbs, for example, may remain unaltered. This is what led early naturalists to conclude that species were related: although dolphins have fins and bats have wings, if you look at the anatomy of the limb you quickly realize that the same building blocks are present. The same bones seem to have been used and reshaped in different ways, leading to different external appearence, but with the internal configuration remaining the same. In contrast, insect legs have a very different constitution (also not being calcified like vertebrate bones are). And so, we may posit that traits that may be less malleable by selection may be more reliable indicators of shared ancestry. [Take an example with a few traits and ask what may be best candidates.] Of course, in practice it is difficult to completely rule out that a trait may look similar because it has experienced similar selective conditions. 

* So which traits do we choose if there is no clear cut answer to whether they might be prone to homoplasies? [Take the platypus for example.] This has bugged systematists (the people that reconstruct relationships between species) for a while, and while the body shape or wing examples have been well understood, sometimes it is much more difficult to get a conclusive answer. For example, there has been debate for a long time over where whales branch within mammals, with the shapes of different bones being used a criteria, and different bones telling different stories. So who do we believe?

* Sometimes, there is wisdom in the crowds. And because evolving lineages have carried with them a number of co-evolving traits, maybe looking at many traits together, and try to identify some majority consensus, could help infer the "real" phylogenetic history of a clade. So, some people have compiled large databases of lots and lots of traits for multiple species, and that helped, but when it comes to number of characters, nothing beats DNA sequences. Genetic sequences can be billions of bases long, and can be sequenced for relatively cheap now (not necessarily always the whole genome, but  bits of it already contain lots of bases). And just like anatomical feeatures, genetic sequences (alsos aminoacid sequences actually) do contain information about phylogenetic relatedness. We use DNA tests to establish paternity for example. We also commonly say that 99%+ of our genetic material is identical with chimps. Bottom line: sequence similarity can also be used as a criterion to infer evolutionary relatedness between species. Except that now we are not restricted to 2, 4, 5, or even 50 characters (which would be the case if focusing on bone shape for example), but we have immediattely access to thousands. Ofc we have to be careful here, because just lke any trait, sequences may be similar not by descent, but by homoplasy. Remember the example with sequences randomly evolving to thhe same base in different lineages. We just hope that if we have lots of different,, relatively independent traits (different base pairs), the signal carried by those that are identical by descent will prevail over those that are homoplasies, especially if those homoplasies are random (e.g. not driven by selection).

* So how does that look like? Well, if each base pair is a trait, then each base pair can tell us its own story about what the phylogeny of the species may have looked like, if we are to believe that the similarities we see are indeed due to sharred ancestry. We might be wrong and some will be homoplasies, but we don't know which ones so let's wait until we have the full consensus. What we end up with is many "gene trees", and from those we infer the "species tree" that is the best depiction of evolutionary relationships that we could come up with. And so that means, that the final tree is a hypothesis about which historical scenario has happened --- back to this later.

* Now in practice, there are things to consider. For example, like we said, homoplasy also is a thing at the sequence level. This may be due to pure chance or due to selection. Hence, just like for "external" phenotypes, we may want to use sequences that are less affected by selection than others. And note: it is not because sequences are "internal" to the organism that they are not subject to selection: the products of genes, for example, may affect the phenotype and affect the representation of teh underlying sequence in tthe populatiton (but that's population genetics). Some sequences code for very important proteins or functions. Others code ffor nothing (a lot of the genome of multtticellular organisms is like that actully). And so, one might expect the "neutral" sequences to slowly accumulate mutations though time in diverging lineages, to the point that the amount of gentic difference can become an indicator of phylogenetic relationship, perhaps more than the amount of sequence divergence at sequences that are under selection. This is important because it alloes not only to reconstruct phylogenetic relationships, but also to estimate the time since divergence. The idea is that sequence divergence is proportional to time apart and so if we know at hich rate subsitutions [and e can sho that this is equal to the muation rate for neutral regions] accumulate we can in principle date the nodes in our phylogney.

* Talk about time calibration and fossils. Maybe show a comparison of a sequence divergence tree and an ultrametric one.

* And yes, this implies that different sequences, or portions of the genome, evolve (i.e. accumulate substitutions) at different speeds: some locations on the chromosomes are more prone ttto mutate, some types of sequences aare more rapidly evolving than others. Sequences that have important functions usually evolve slowly, they are conserved: that is because most of the mutations that hit them disappear quickly from the population, being deleterious. And the speed of evolutionary change affects the probability that homoplasies will arise. [Sho example.] hat does that mean? The implication is that the choice of hich region of the genome to use to reconstruct a phylogeny depends on the temportal resolution you are going for. Sure, neutral regions are perhaps preferable because no bias due to seleciton, but the speed at hich they accumulate mutations makes them impractical for very deep phylogenies (e.g. LUCA and 16S rRNA) --- at those scales homoplasies are guarantieed to arise and similarity becomes a poor predictor of shared ancestry. For large time scale we therefore use stuff that evolves slowwly ---e.g. just taking the exons of genes. Neutral markers might be good for divergence beteen recently diverged species or maybe populations within the same species. You basically have to pick  the type of sequence that matches the resolution of the phylogeny you are unraveling (so you better have some idea of that resolution ofc).

* In practice, reconstructing a phylogeny is a complicated process. You need to align the sequences properly (indels and synteny issues make it difficult). And then, many different algorithms exist. Modern techniques do not simply do as we did in the previous example --- in which we actually used the (now outdated) maximum parsimony criterion. That criterion invokes Okham's razor by saying that for a given distribution of a characcter (a base pair) among taxa, the scenario that requires the fewest assumptions (here substitutions) to arrive at the final patttern is probably the correct one. While this alfgorithm has been used a lot (and is still being used for non-sequence data), it has now fallen out of favor in most of phylogenetics, because of issues regarding (again) homoplasies. Homoplasies will make you infer fewer substitutions than there actually were. And as we saw, a couple of factors can inflate the probability of homoplasies: genomic regions that evolve fast, or genomic regions experiencing similar selective pressures in different taxa. Another factor increasing that risk is that not only substitution rates are different for tidfferent portions of the gebome,, thhey also vary from taxa to taxa. Some taxa are "fast evolving", and Felsenstein shoed that this could cause a "long branch attraction" artifact. 

* As I said, there are now more modern methods that try to account for those pitfalls in vvarious ways. But mostly, state-of-the-art methods use probability ttheory and are based on models of how things evolve. They say: let's say there are certain rates for the substitutions that can happen to a specific base pair, and that those subsitutions are a stochastic process (the rates then represent the probability that the substitutions occur). Now if the phylogeny that gave rise to the data we have (so the tree we are trying to recosntruct) is e.g. like [this] and the rates have [these] values, we shouldd be able to calculate the probability that the alignemtn we observe occurred. That probbaility is called the likelihodd of the data fiven the model and the tree. And if wwe have a formula that give us that value, then we can make use of likelihood maximization procedures to (computationally) find the combination of rate values and tree that best explain our alignment --- that is, that most likely gave rise to the data we see today. This would be extemely tedious to do by hand, but what we basically do in most of phylogenetic reconstruction is let the computer try lots and lots of combinations of these rates and of phylogenetic relationships (i.e. topologies of the tree) and come up with those that give the highest likelihood (the retained values and tree are called maximum likelihood estimates). This is a technique that is vvery idespread in statistics, henever one needds to fit a model with some parameters to some data. Here,, the model is not just numbers bbut also a tree. Ofc there are lots of technicalities in ho ww to efficiently explore the space of possible combinations and come up wwith th best estimates, but this is ot of scope here. The key difference beteen likelihood based method and parsimony is that likelihood uses probabilities, by assuming a stochastic underlying model. That means tht it accounts for uncertainty --- it wweighs possibilities based on ho probable they are given a model, and do not necessarily say "the solution with the fewest steps is always the best" (say this better).

* Now the actual most-used methods nowwadays are bayesian. Those are likelihood based, but they do not actually maximuse the likelihood, they maximize the posterior probability (out of scope here I would say), wwhich is stlightly different but the core spirit is the same: try ots of combinations of parameters of an underlying model and see what fits best. Likelihood maximization and Bayesian methods are therefore methods of  model fittin --- and they are used for lots and lots off applications in statistical sciences. Maybe show examples of posterior trees and consensus.

* [Say something about fixing rates of transitions and transversions or not?]

* [Say something about phylogenomics.]

* Now we move on to applications of phylogenies. That's gonna be mini videos for the practicals.

* Example 1: Phylogenetic regression. 

* It is typical when doing statistics to collect data points and test hypotheses based on those points. Is there a relationship betwween X and Y? Are two or more groups significantly different wwith respect to metric Z? To answer those questions we use statistical tests, like linear regressions and ANOVAs. Those statistical tests aim at telling us if there is a strong signal in our data, i.e. if we can conclude that the effect we see is stronger than expected bby chance. Now statistical tests come wwith assumptions about the data they are applied to, and if the assumptions are not met, the results of the test may not be trustworthy. [Show an example wwith an outlier for example.] Assumptions change from test to test, but one that is very common is that of statistical non-independence. Picture this: [showw an example wwith members of the same family]. Now, non-independence introdduces pseudo-replication: the fact that different data points are not exactly replicates. And that is a problem because we may falsly infer stuff if we consider them independent. Many so-called comparative studies use species (or taxa, or populations) as data points. But wwe know that different species have different degrees of relatedness with every other one. Some species will be more closely related than others, and the same as in the example, that might mess things up. Fortunately, thanks to phylogenetics it is possible (ith tthe tree) to tell who is more related to whom, and use that information to counteract this pseudo replication problem. Indeed, if we know how close each spceies is from every other, we can take that to weigh dowwn or up the signal we see in the data. There are methods to do that, like PAgel's lambda and Felsenstein's independent contrasts, and those are implemented in popular statistical softwware like R. In this practical you will learn to perform a phylogenetic regression, i.e. a regression that answers a question using spceies as data points, taking into account the phylogenetic non-independnece of the species (and so asking: after we remove the confounding effect of relatedness, is there a pattern or not?). [Maybe show an example here we just replace family members wwith species.]

* Example 2: Ancestral character reconstruction

* Evolutionary biology is a lot like a crime scene reconstruction: wee try to reconstruct the genealogical relationships between species, and to infer what happenedd in the past, most of the time only using clues we have availabel in the present. One question that many researchers ask is: what did the ancestors of this or that spceies look like? Where did they live? What did they likely feed on? Of course fossils help a lot there [we know that the mrrca between humans and chimps probably looked more like a chimp than like a human], but again, in their absence it may still be possible to get insights in those questions using extant spceies onlyy. For this we turn to ancestral character reconstruction methods. As their name suggests, those are methods designed to reconstruct the evolutionary history of a trait, typically along the branches of a phylogeny. Here is how it works: provided we have a phylogeny (that is an important bit) and data about a given trait in the different spceies of that phylogeny at present (so at the tips), we can statistically infer back in time what the ancestors may have looked like. For this wwe use relatively simple models of trait evolution. In the simplest one called Brownian motion, e assume that traits change like prticles diffuse in space, randomly and stochastically throughh time, but more often little than a lot. Now just like we could used likelihood procedures to figure what is the most likely phylogenetic history that explains a given sequence allignment when we reconsctructed our tree, now we can ask, since wwe have a model from hich we can derive a formula for its likelihood, what are the parameters of this Brownian motion model that best explain the distribution of trait values we observed at present, given the phylogenetic history as constraint. If we do that, and sparing the details, we can get estimates (wwith error margins of course) of wwhat the traits at the nodes may have looked like. Of course, how much to trust these reconstructions boils down to howw much you trust the assumptions (again, after all those models are statistical models because we fit them to data) of the model you used. BM may be a bit too simplistic in many applications, and so you may wwant to go for a model with fancier assumptions, like OU. [Explain that.] Bear in mind, however, that there is a tradeoff betwween the complexity of a model annd hether it can be fitted to data --- if the model is too complicated, it may be impossible to find the formula for its likelihood. Which model to trust then? Well, one thing you can do is fit multiple ones to your data and use goodness-of-fit criteria (such as AIC, which compares the likelihood but penalizes for model complexity) to tell you wwhich model fits your data best, and so, it may be yourr data that tell you that BM is actually a better explanation for the pattterns you observe than OU. (How do I fit in discrete characters? Maybe see what Juan Jos√© comes up with.)  

* Example 3: Dectection of hybridization and introgression

* In the lecture on speciation we will have learned that species are typically reproductively isoalted from each other. However, that is not always the case and there are exceptions. Actually, some researchers aregue that hybridization betwween spceies is the rule mmore than just an exception. What do they base that on? Well, we can detect hybridization between the genomes of two separate species using signatures in their DNA, and specifically, using phylogenetics. Remember, we said that when using genetic data, what we reconstruct is gene trees: phylogenies that represent the most likely history one gene at a time. Then, we find a way (we didn't go into detail about that) to infer a consensus species tree based on what all the separate genes have to say. We said that this was to account for the fact that some genes wwill show homoplasies, so we want to rely on the majority consensuus. But sometimes different genes tell different stories, not because one really represents the history of the species and the other one suffers from homoplasies, but because they both underwent different evolutionary routes. This happens when there is hybridization: after hybrids reproduce with members of either of their "parent" spceies, foreign genes ill be introduced in a population, but because of recombination, some will be lost and some will persist. So, only some genes will introgress into another species. Those genes have undergone a different evolutionary history than the rest of the genome, because they come from a different species. The gene tree we reconstruct for them represents the true evolutionary history they've been through, and will be different from that of other genes, without homoplasies. Using sequencing of many many genes, genomists use the discrepancies in gene trees among different portions of the genome to identify which ones mmay have introgressed between species, using a classic test called ABBA-BABA test. [Explain howw it wworks.] This served, for example,, to identify that about 2% of the genome of caucasian people comes from Neanderthal, and a similar small portion of the genomes of asian and polynesian people come from Denisovans, two extinct spceies of humans that were hybridizing with our ancestors tens of thousands of years ago! (Mention incomplete lineage sorting as source of discrepancy bteweeen gnes? Or is that too much??)

* Example 4: Phylodynamics

* So far we have seen how we can use the shape of a phylogenetic tree to ask more precise questions about other things, such as gene flow beteen species or ancesttral characters. But the shape of the phylogeny may itself be a topic of enquiry --- that is the field of phylodynamics (now that name is mostly used in epidemiology, wwhere the phylogenies are those of pathogens like covid, but the prrinciples apply to any kind of tree). Why do we want to explain the shape of phylogenies? That is because they are not trivial and random! For example, most reconstructed phylogenies are unbalanced: the repartition of species within clades is very asymmetrical. Or, most phylogenies exhibit a diversification slowdown, which we can see if we plot the so-called ltt plots. Those patterns are the result of a balance beteen speciation adn extinction (both of which can be estimated from molecular phylogenies, and that is nother part of phylodynamics). What determines those is not very well understood however (there are theories of course). Phylodynamics is all about finding and testing those theories, in the hope to understand the forces that shape phylogeniies, and the factors that influence speciation and extinction rates. In this tutorial we explore in more detail [ddd? bisse?].

* (Make mini-videos on these different applications.)

